org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/CorePlugin.java

```diff
@@ -187,7 +195,30 @@ private UserNotification createUserNotification() {
         return new ConsoleUserNotification();
     }
 
+    /*
+     * https://resheim.net/2008/10/connecting-through-proxy.html
```

Comment:
I'm not certain how to attribute work to other people, especially to what seem to be tutorials. Any suggestions are appreciated.